doc/user_guide/display_frontends.rst

```diff
@@ -179,57 +184,83 @@ If you are using a dashboarding package that is not listed here, please `open an
 
-    import altair as alt
+Example usage::
 
-    # load a simple dataset as a pandas DataFrame
-    from vega_datasets import data
-    cars = data.cars()
+    alt.renderers.enable("png", scale_factor=2, ppi=144)
 
-    chart = alt.Chart(cars).mark_point().encode(
-        x='Horsepower',
-        y='Miles_per_Gallon',
-        color='Origin',
-    ).interactive()
 
-    chart.show()
+.. _display-browser:
+
+Browser Renderer
+~~~~~~~~~~~~~~~~
+To support displaying charts with interactive features in non-browser based environments,
+the ``"browser"`` renderer automatically opens charts in browser tabs of a system web browser.
+
+The ``"browser"`` renderer is enabled like this::
+
+    alt.renderers.enable("browser")
+
+
+The ``"browser"`` renderer supports the following keyword argument configuration options:
+
+- The ``using`` argument may be used to specify which system web browser to use. This
+  may be set to a string to indicate the single browser that must be used (e.g. ``"safari"``),
+  or it may be set to a list of browser names where the first available browser is used. See the
+  documentation for the `webbrowser module`_ for the list of supported browser names. If not
+  specified, the system default browser is used.
+- The ``offline`` argument may be used to specify whether JavaScript dependencies should
+  be loaded from an online CDN or embedded alongside the chart specification. When ``offline``
+  is ``False`` (The default), JavaScript dependencies are loaded from an online CDN, and so
+  an internet connection is required. When ``offline`` is ``True``, JavaScript dependencies
+  are embedded alongside chart specification and so no internet connection is required. Setting
+  ``offline`` to ``True`` requires the optional ``vl-convert-python`` dependency.
+- The ``port`` argument may be used to configure the system port that the chart HTML is served
+  on. Defaults to a random open port.
+
+Limitations:
+
+- The ``"browser"`` renderer sets up a temporary web server that serves the chart exactly once,
+  then opens the designated browser pointing to this server's URL. This approach does not require
+  the creation of temporary HTML files on disk, and it's memory efficient as there are no long-lived
+  web server processes required. A limitation of this approach is that the chart will be lost if the
+  browser is refreshed, and it's not possible to copy the chart URL and paste it in another browser
+  tab.
+- When used in IPython-based environments, the ``"browser"`` renderer will automatically open the
+  chart in the browser when the chart is the final value of the cell or command. This behavior is not
+  available in the standard ``python`` REPL. In this case, the ``chart.show()`` method may be used to
+  manually invoke the active renderer and open the chart in the browser.
+- This renderer is not compatible with remote environments like Binder or Colab.
```

Comment:
Could we use links also for "Binder" and "Colab" so that it's consistent with Ipython and Spyder above? 

```suggestion
- This renderer is not compatible with remote environments like `Binder`_ or `Colab`_.
```